CHANGELOG.md

```diff
@@ -22,6 +22,9 @@ All notable changes to this project will be documented in this file.
 - trino: Add 469 ([#993]).
 - trino-cli: Add version 469 ([#993]).
 - trino-storage-connector: Add version 469 ([#996]).
```

Comment:
~I think we should remove that entries again, WDYT?~

I see you have removal entries further down. I personally would argue it's better to not confuse users with release-internal stuff, but I let you decide :)

Reply:
Yeah, I wasn't sure either, but is is a "code changelog", not release notes, so I'd keep it as is.